curvefs/src/client/curve_fuse_op.cpp

```diff
@@ -273,6 +276,60 @@ void FuseOpGetAttr(fuse_req_t req, fuse_ino_t ino, struct fuse_file_info *fi) {
     fuse_reply_attr(req, &attr, g_fuseClientOption->attrTimeOut);
 }
 
+int addWarmup(const std::string&& type, const std::string& path) {
```

Comment:
CamelCase, and why first arguemtn is a const rvalue reference?

Reply:
> CamelCase, and why first arguemtn is a const rvalue reference?

bug，I wrote  PutWarmTask and FetchDentryEnqueue  before, used rvalues.